src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlJson.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+    internal class SqlJson
```

Comment:
Can this file be moved to src (shared) folder? Look for files in path: `src\Microsoft.Data.SqlClient\src\...`

Reply:
Will add it later under SqlTypes as part of implmenting SqlJson class. I have removed it for now.